app/src/main/java/com/yacgroup/yacguide/utils/VisualUtils.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2023 Axel Paetzold
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.yacgroup.yacguide.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.ContextCompat
+import com.yacgroup.yacguide.R
+
+class VisualUtils(
```

Comment:
The reason for having this class is:
When creating the RecyclerView list items, you'd need to call ContextCompat methods (like `ContextCompat.getColor()` e.g.) for every item. Using `VisualUtils`, the according values are set on activity creation and no method calls are needed when iterating over the item list.